packages/xrpl/src/client/index.ts

```diff
@@ -564,6 +564,50 @@ class Client extends EventEmitter {
   /**
    * Tells the Client instance to connect to its rippled server.
    *
+   * @example
+   *
+   * To use the connect() method, you first need to create a new Client object using the new Client() constructor:
+   *
+   * ```ts
+   * const { Client } = require('xrpl')
+   * const client = new Client('wss://s.altnet.rippletest.net:51233')
+   *
+   * This creates a new Client object and specifies the URL of the server to connect to. You can replace the URL
+   * with the URL of any other server that supports the XRP Ledger.
+   *
+   * After you've created the Client object, you can call the connect() method to connect the client to the server:
+   *
+   * ```ts
+   * client.connect().then(() => {
+   *  console.log('Connected to server')
+   * }).catch((error) => {
+   *  console.log(`Failed to connect to server: ${error}`)
+   * })
+   * ```
+   *
+   * This will connect the client to the server and log a message to the console when the connection is established.
+   * If there's an error connecting to the server, the catch() method will log an error message to the console.
+   *
+   * It's important to note that the connect() method returns a promise, which resolves when the connection is
+   * established and rejects if there's an error connecting to the server. This means you can use the async/await
+   * syntax to wait for the connection to be established before executing any further code:
+   *
+   * ```ts
+   * async function connectToServer() {
+   *   try {
+   *     await client.connect()
+   *     console.log('Connected to server')
+   *   } catch (error) {
+   *     console.log(`Failed to connect to server: ${error}`)
+   *   }
+   * }
+   *
+   * connectToServer()
+   * ```
+   *
+   * In this example, the connectToServer() function uses the async/await syntax to wait for the connection to
+   * be established before logging a message to the console.
+   *
```

Comment:
Simplifying to just use `await` syntax since that's preferred, and only commenting on the usage parts instead of re-describing the code.

```suggestion
   * Client.connect() establishes a connection between a Client object and the server.
   *
   * ```ts
   * const { Client } = require('xrpl')
   * const client = new Client('wss://s.altnet.rippletest.net:51233')
   *   try {
   *     await client.connect()
   *     console.log('Connected to server')
   *   } catch (error) {
   *     console.log(`Failed to connect to server: ${error}`)
   *   } 
   * ```
   * If you open a client connection, be sure to close it with `await client.disconnect()` 
   * before exiting your application.
```